pkg/reconciler/buildrun/resources/sources.go

```diff
@@ -31,14 +31,15 @@ func isLocalCopyBuildSource(
 
 	// Note: In v1alpha1 we will append all sources across builds and buildruns, and then return
 	// the first source of type Local.
```

Comment:
The comment becomes an orphan by your change, nonetheless we think this comment can be removed anyway. Could you amend that in your change?

Reply:
Yeah sure @HeavyWombat . will do it ASAP!